app/javascript/src/components/Common/DeletePrompt/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+import { Modal, Typography, Button } from "@bigbinary/neetoui/v2/";
+import PropTypes from "prop-types";
+
+const DeletePrompt = ({
+  showPrompt,
+  onDelete,
+  onCancel,
+  title = "Delete Item",
+  message = "Are you sure you want to delete this?",
+}) => (
+  <Modal isOpen={showPrompt} onClose={onCancel}>
+    <Modal.Header>
+      <Typography style="h2">{title}</Typography>
+    </Modal.Header>
+    <Modal.Body>
+      <Typography style="body2" lineHeight="normal">
+        {message}
+      </Typography>
+    </Modal.Body>
+    <Modal.Footer className="space-x-2">
+      <Button label="Continue" onClick={onDelete} size="large" />
+      <Button style="text" label="Cancel" onClick={onCancel} size="large" />
+    </Modal.Footer>
+  </Modal>
+);
+
+DeletePrompt.propTypes = {
+  showPrompt: PropTypes.bool.isRequired,
+  onDelete: PropTypes.func.isRequired,
+  onCancel: PropTypes.func.isRequired,
+  title: PropTypes.string,
+  message: PropTypes.string,
+};
+export default DeletePrompt;
```

Comment:
We don't need this. Please use [Alert](https://neetoui-v2.netlify.app/?path=/story/overlays-alert--alert-story) from neetoUI.

Reply:
Did this as there was no way to change the style of button in the neetoUI alert to the one shown in the Figma design.